sycl/doc/extensions/SPIRV/SPV_INTEL_non_constant_addrspace_printf.asciidoc

```diff
@@ -0,0 +1,164 @@
+:extension_name: SPV_INTEL_non_constant_addrspace_printf
+:capability_name: NonConstantAddrspacePrintfINTEL
+:capability_token: 6411
+
+{extension_name}
+================
+
+
+== Name Strings
+
+{extension_name}
+
+== Contact
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/intel/llvm
+
+== Contributors
+
+- Leonid Pauzin, Intel +
+- Dmitry Sidorov, Intel +
+- Nikita Kornev, Intel +
+
+== Notice
+
+Copyright (c) 2022 Intel Corporation.  All rights reserved.
+
+== Status
+
+Working Draft
+
+This is a preview extension specification, intended to provide early access to a
+feature for review and community feedback. When the feature matures, this
+specification may be released as a formal extension.
+
+
+Because the interfaces defined by this specification are not final and are
+subject to change they are not intended to be used by shipping software
+products. If you are interested in using this feature in your software product,
+please let us know!
+
+== Version
+
+[width="40%",cols="25,25"]
+|========================================
+| Last Modified Date | 2022-01-13
+| Revision           | A
+|========================================
+
+== Dependencies
+
+This extension is written against the SPIR-V Specification,
+Version 1.5, Revision 5.
```

Comment:
The latest available revision is 6. Should we update to the latest reversion?

Reply:
Thanks, applied in 906a4cd